plugin/core/sessions.py

```diff
@@ -257,6 +257,56 @@ def get_initialize_params(variables: Dict[str, str], workspace_folders: List[Wor
             },
             "codeLens": {
                 "dynamicRegistration": True
+            },
+            "semanticTokens": {
+                "dynamicRegistration": True,
+                "tokenTypes": [
+                    "namespace",
+                    "type",
+                    "class",
+                    "enum",
+                    "interface",
+                    "struct",
+                    "typeParameter",
+                    "parameter",
+                    "variable",
+                    "property",
+                    "enumMember",
+                    "event",
+                    "function",
+                    "method",
+                    "macro",
+                    "keyword",
+                    "modifier",
+                    "comment",
+                    "string",
+                    "number",
+                    "regexp",
+                    "operator"
+                ],
+                "tokenModifiers": [
+                    "declaration",
+                    "definition",
+                    "readonly",
+                    "static",
+                    "deprecated",
+                    "abstract",
+                    "async",
+                    "modification",
+                    "documentation",
+                    "defaultLibrary"
+                ],
+                "formats": [
+                    "relative"
+                ],
+                "requests": {
+                    "range": True,
+                    "full": {
+                        "delta": True
```

Comment:
Same here.

Reply:
Yes, `textDocument/semanticTokens/full/delta` is not implemented.